src/llamafactory/chat/vllm_engine.py

```diff
@@ -77,7 +77,7 @@ def __init__(
             "max_lora_rank": model_args.vllm_max_lora_rank,
         }
 
-        if model_args.visual_inputs:
+        if model_args.visual_inputs and model_args.visual_inputs_type == "vision_tower":
```

Comment:
Hi @marko1616, thank you for these commits. I was wondering here with this condition, does it mean that we do not need to do anything when it's not llava style models (i.e. `visual_inputs_type` not being `vision_tower`)?

Reply:
There is a good news that i have some more compute device.I will fix this within about 1 week?

Reply:
Those two models(glm4v and qwenvl) seems havn't completed fully support form now.I will raise a `NotImplementedError` for this two models and give a support for phi3v.Really thank for pointing this out.
Here is some related infos.
https://github.com/vllm-project/vllm/pull/5358
https://github.com/vllm-project/vllm/pull/5710